lms/static/js/ccx/collection/schedule_collection.js

```diff
@@ -0,0 +1,269 @@
+var define = window.define || RequireJS.define;  // jshint ignore:line
+
+define(
+  [
+    'backbone',
+    'underscore',
+    'gettext',
+    'moment',
+    'js/ccx/model/schedule_model'
+  ],
+  function (
```

Comment:
We use a coding convention of 4 spaces per indent for both JavaScript and Python.

https://github.com/edx/edx-platform/wiki/Javascript-standards-for-the-edx-platform

If you haven't seen this page before, please review it:

https://openedx.atlassian.net/wiki/display/TNL/Code+Review+Guidelines+and+Gotchas